docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,53 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+The project can be enabled to be configured by using `MonitorService` resources from
+the Prometheus Operator. Following the steps to do this setup.
```

Comment:
How about "Prometheus Operator, by following these steps:"?

Reply:
IHMO the text has a hall to be improved but just Prometheus Operator, ... shows not too much clear.  
I changed let's see if you agree :-)